spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Merge.scala

```diff
@@ -116,25 +120,15 @@ class Merge[T: ClassTag](
 
   override def doBuild(inputShape: Shape): AbstractModule[Tensor[T], Tensor[T], T] = {
     val input = inputShape.toMulti()
-    val model = TSequential[T]()
-    val parallel = ParallelTable()
-    var i = 0
-    while(i < layers.length) {
-      val tlayer = layers(i) match {
-        case k: KerasLayer[_, _, T] => k.labor
-        case t: AbstractModule[Activity, Activity, T] => t
-      }
-      parallel.add(tlayer)
-      i += 1
-    }
-    model.add(parallel)
     val seq = TSequential[T]()
     val layer = mergeMode match {
```

Comment:
rename to merge_layer as its unclear together with parameter "layers"

Reply:
Renamed.